utility/src/main/java/org/oppia/android/util/parser/GlideImageLoader.kt

```diff
@@ -14,7 +14,7 @@ import org.oppia.android.util.caching.CacheAssetsLocally
 import javax.inject.Inject
 
 /** An [ImageLoader] that uses Glide. */
-class GlideImageLoader @Inject constructor(
+open class GlideImageLoader @Inject constructor(
```

Comment:
Prefer injecting this class and calling directly to it rather than inheriting (or put another way: prefer composition over inheritance when extending functionality).

Reply:
Ok, done.